src/rule-tester.ts

```diff
@@ -23,8 +38,16 @@ export function createRuleTester(options: RuleTesterInitOptions): RuleTester {
       defaultConfigs.unshift(inlineConfig)
   }
 
+  const defaultFilenames = {
+    js: 'file.js',
+    ts: 'file.ts',
+    jsx: 'react.jsx',
+    tsx: 'react.tsx',
```

Comment:
```suggestion
    jsx: 'file.jsx',
    tsx: 'file.tsx',
```

Wouldn't it better just having `file` and be more natural? Any reason `react` is needed?

Reply:
That's just how typescript-eslint does it for their tester so I just copied them.